Sources/ComposableArchitecture/SwiftUI/Alert.swift

```diff
@@ -129,7 +129,15 @@ public struct AlertState<Action> {
 
   public struct Button {
     public var action: Action?
+    public var animation: Animation??
     public var type: `Type`
+    
+    public func send(_ action: Action, withAnimation animation: Animation?) -> Self {
+      var button = self
+      button.action = action
+      button.animation = animation
+      return button
+    }
```

Comment:
Instead of this single `send`, we could overload each of the helpers below with a version that takes an `Animation?`. This way the API would match how `ViewStore.send` has an overload that takes an `animation` parameter. It'll be a bunch of overloads, but shouldn't be so so bad.